### ISSUE TITLE
Google Play Store release

### ISSUE THREAD

**Issue by hufman** [hufman/AndroidAutoIdrive#212]
With the improved self-guided instructions in the app, it should be easy enough for random users to install the app without needing to refer to the website. This unlocks the ability to post the app to the Play Store.

First blocker: What to name the Developer that it's posted under?

**Comment by Sven31415** [comment]
I recommended this App to every BMW/Mini driver I know and I think everybody should use/have it. 
And I like the idea of having it on the store, so more people can use it.
However I'm a bit worried that BMW won't like this, and maybe find a way to prevent people from using it. So maybe to much attention isn't that good? 
On the other hand, if a solution is found to connect to the iDrive which doesn't need the connected app as a tunnel, I wouldn't be worried anymore. Because then they couldn't do anything

**Comment by hufman** [comment]
I feel pretty confident that it should be fine. As a prime example, Bimmercode is available on both Apple and Android app stores, and it does much naughtier things and charges a fee for it.
This app should be even more acceptable that Bimmercode: It is free, it doesn't unlock any features that you would otherwise pay BMW for and is sandboxed off to not injure the car, and BMW has deprecated the protocol anyways and isn't competing with it.

**Comment by fookyuanloh** [comment]
Perhaps you steer clear of using "BMW" or any of their trademark names

**Comment by bogdan-calapod** [comment]
Let me know if there is any artwork/screenshots needed for the listing that I can help with 😀

**Comment by berseker** [comment]
I think that in this occasion (opening the app to the "big audience") you should evaluate to add in the "connection tab" a link to ( or directly all the tips mentioned in) these pages

https://hufman.github.io/AndroidAutoIdrive/faq.html
https://hufman.github.io/AndroidAutoIdrive/connection

**Comment by hufman** [comment]
I agree, I've been working to incorporate this information into the app itself, in a concise and contextually-relevant way: For example, it'll detect when the phone is in USB Charging mode and prompt to switch to USB File Transfer mode.
Additionally, adding the information to the app allows for being translated, while the webpage is harder to translate.

**Comment by Royalismyrace** [comment]
Maybe "Stay Connected"

**Comment by bogdan-calapod** [comment]
Honestly with the whole situation of BMW discontinuing Connected,  that sounds like a great name !

**Comment by berseker** [comment]
I would like to suggest AndroIDrive   (a fusion between Android & Idrive)

**Comment by fookyuanloh** [comment]
Any updates on this item?

**Comment by hufman** [comment]
No updates currently, I'm working on finishing the 1.3 release before working on this.

**Comment by fookyuanloh** [comment]
Sorry to repost on this topic
Since issue #213 was closed and we have a name and independent github repo., are we releasing it via Playstore yet?

**Comment by hufman** [comment]
There is some gradual progress, I will comment here when it's ready for testing 😃

**Comment by hufman** [comment]
I'm ready to start letting people try installing the app from the Play Store! Please email me with what Google account you'd like me to invite and then I'll add you to the test group and send you the link!

**Comment by fookyuanloh** [comment]
i'm in
sent you an email over h****n@gmail.com

looking for to hear frm you

cheers !!!

**Comment by fookyuanloh** [comment]
Awesome, thank you so much!
Got the invite and installed the app via Google Playstore. 

Tested it on my F30 2016, NBT EVO ID4
Using myBMW app v1.60

Most functions works - using Spotify most of the time

**Comment by Royalismyrace** [comment]
***@***.***

Sent by Outlook för Android<https://aka.ms/ghei36>
________________________________

**Comment by Royalismyrace** [comment]
***@***.***

Sent by Outlook för Android<https://aka.ms/ghei36>
________________________________

**Comment by hufman** [comment]
Email me at hufman+aaidrive@gmail.com with your information to join the invite list!

**Comment by hufman** [comment]
The store listing is now public: https://play.google.com/store/apps/details?id=me.hufman.androidautoidrive

**Comment by berseker** [comment]
My best congrats for this awesome milestone reached!!

**Comment by Royalismyrace** [comment]
Yippi, congrats to you and finally its live

John Hedberg

Sent by Outlook för Android<https://aka.ms/ghei36>
________________________________

**Comment by HaivanJV** [comment]
Amazing milestone! Congrats and thanks for all the hard work!

**Comment by hufman** [comment]
Hmmm I'm not quite sure I understand from this description, can you attach a picture on Github to show what you mean? It sounds like the image that would be shown in the menu (light-colored map, with a black border for not-widescreen mode on a widescreen-supported screen). It should switch to a different screen after clicking through View Full Map and start rendering the full screen map.

**Comment by Royalismyrace** [comment]
Today it works fine?

Regards John Hedberg

Sent by Outlook för Android<https://aka.ms/ghei36>
________________________________

**Comment by hufman** [comment]
Weird! There [haven't been any updates](https://github.com/BimmerGestalt/AAIdrive/actions/workflows/playstore-publish.yml) to the store release.

**Comment by Royalismyrace** [comment]
Probably my idrive got a little 6 shock 🤯 first day. The only thing I probably saw upp in the top left vas the scale bar I now reckon. And the map quest logo was at the same place as on the picture.

Sent by Outlook för Android<https://aka.ms/ghei36>
________________________________

**Comment by hufman** [comment]
Neat! I did move the scalebar recently (9f53c8e) but that should have been in the Play Store build... Anyways, glad it works now!

**Comment by Royalismyrace** [comment]
Yeah unless you have made a self improving program code with a little help from the Borgs... Soon we will be assimilated to BMW 😜😎

Sent by Outlook för Android<https://aka.ms/ghei36>
________________________________

**Comment by hufman** [comment]
I encountered that black screen a few times recently, seemingly when cell/wifi signal was too low at startup, and added a commit today that tries to reload the basemap layer whenever you change windows, which should make it possible to resolve the black screen without restarting the app. Thank you for your testing and feedback!
